zerver/lib/message.py

```diff
@@ -344,3 +344,169 @@ def render_markdown(message, content, realm=None, realm_alert_words=None, messag
         message.is_me_message = Message.is_status_message(content, rendered_content)
 
     return rendered_content
+
+def huddle_users(recipient_id):
+    # type: (int) -> str
+    display_recipient = get_display_recipient_by_id(recipient_id,
+                                                    Recipient.HUDDLE,
+                                                    None)  # type: ignore
```

Comment:
Should we not put in here the mypy type definition as `Union[Text, List[Dict[str, Any]]]`?